bot/handler/callbacks.py

```diff
@@ -41,8 +42,11 @@ async def button(update: Update, context: ContextTypes.DEFAULT_TYPE) -> int:
                 case BotStates.RECEIVE_SCREENSHOT:
                     await confirm_screenshot(update, context)
                 case _:
-                    report.create_report(context.user_data["report"])
+                    report_response = report.create_report(context.user_data["report"])
                     track_user_event(update, context, Event.REPORT_CREATED)
+                    print("report_response",report_response)
+                    embeddings.insert_embedding(context.user_data["embedding"], report_response[0]["id"])
```

Comment:
if you use `r, _ = `  you could do:
```suggestion
                    embeddings.insert_embedding(context.user_data["embedding"], r["id"])
```